schema/mysql/v57/temporal/schema.sql

```diff
@@ -79,7 +79,7 @@ CREATE TABLE buffered_events (
 
 CREATE TABLE tasks (
   range_hash INT UNSIGNED NOT NULL,
-  task_queue_id VARBINARY(255) NOT NULL,
+  task_queue_id VARBINARY(272) NOT NULL,
```

Comment:
I know 255 but who is 272?

Reply:
255 for TaskQueueName originally + 16 byte NamespaceId + 1 byte for a uint8 casted TaskQueueType